pkg/services/preferences/store/preferences_store.go

```diff
@@ -0,0 +1,100 @@
+package preferencesstore
+
+import (
+	"context"
+	"time"
+
+	"github.com/grafana/grafana/pkg/models"
+	"github.com/grafana/grafana/pkg/services/sqlstore"
+	"github.com/grafana/grafana/pkg/setting"
+)
+
+type StoreImpl struct {
+	SqlStore sqlstore.Store
+	Cfg      *setting.Cfg
+}
+
+// TODO : Get and Set store methods
+type Store interface {
+	// TODO adjust the methods to Get/Set methods move logic to service
+	Get(context.Context, *models.GetPreferencesQuery) (*models.Preferences, error)
+	GetDefaults() *models.Preferences
+	List(ctx context.Context, filter string, params []interface{}) ([]*models.Preferences, error)
```

Comment:
You can remove the variable names here to make it consistent with the rest of the functions.